styles/products.css

```diff
@@ -89,7 +87,7 @@
   width: 25px;
```

Comment:
a suggestion : use 1.5 rem if possible in place of 25px

Reply:
Resolved